en/importer/materials/fbx-materials.md

```diff
@@ -0,0 +1,122 @@
+# FBX Smart Material Conversion
+FBX Smart Material Conversion in Importer is a feature of model importer, which can map some standard materials in DCC tools to Cocos Creator's built-in materials, and restore the effects of materials in DCC tools.
+* The following materials are supported:
+
+|         | Phong               | PBR               |
+|---------|---------------------|-------------------|
+| 3ds Max | Standard(legacy)    | Physical Material |
+| Blender | None                | Principled BSDF   |
+| C4D     | Standard            | None              |
+| Maya    | Lambert/Blinn/Phong | Standard Surface  |
+ 
+* Here is the comparison of maya standard surface in Maya and Cocos Creator:
```

Comment:
Here is the comparison of a model using standard surface viewed in Maya and viewed in Cocos Creator after import.